Orange/widgets/visualize/owmosaic.py

```diff
@@ -518,6 +533,30 @@ def send_selection(self):
     def send_report(self):
         self.report_plot(self.canvas)
 
+    def get_values(self, data):
+        attr_list, variable_color = self.get_attr_list()
+        if variable_color == '' or None:
+            if len(data.domain.class_vars) > 0:
+                color_var = data.domain.class_vars[0]
+            elif data.domain.attributes[0]:
+                color_var = data.domain.attributes[0]
+            else:
+                color_var = data.domian.metas[0]
```

Comment:
If the first meta attribute is a string, nothing happens. But I am still testing.